FrontEndApp/autodl-platform/.env.example

```diff
@@ -6,3 +6,7 @@ FASTAPI_URL=http://localhost:8080
 
 # upload to github
 GITHUB_APP_CLIENT_ID=abcdoadsnfioasndf343
+
+# razorpay api keys (required)
```

Comment:
```suggestion
# razorpay api keys
```

Reply:
Ok,@ADI10HERO 
I have removed the (required) comment